src/object-identifier.scss

```diff
@@ -0,0 +1,61 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-object-identifier;
+
+.#{$block} {
+  @mixin fd-object-identifier-title-base {
+    @include fd-reset();
+
+    color: var(--sapList_TextColor);
+  }
+
+  $fd-object-identifier-title-font-size: var(--sapFontLargeSize);
+  $fd-object-identifier-title-font-weight: bold;
+
+  &__title,
+  &__title--bold {
+    @include fd-object-identifier-title-base();
+  }
+
+  &__title--link {
+    @include fd-reset();
+    @include fd-link();
+    @include action-cursor();
+  }
```

Comment:
If this is used as a modifier class it should not have a reset as it will reset the styling from the BLOCK and the ELEMENT.

Reply:
fixed